src/tests/Microsoft.PowerFx.Core.Tests/IntellisenseTests/SuggestTest.cs

```diff
@@ -594,5 +612,19 @@ public override int GetHashCode()
                 return 1;
             }
         }
+
+        // Helper to set Engine.SupportedFunctions.
+        private class Engine2 : Engine
```

Comment:
We don't really want products to need to derive from an Engine just to update the functions. Let's consider another pattern.